src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private TaskManager manager = new TaskManager();
+
+    public void exit() {
+        System.out.println("------------------------------------");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("------------------------------------");
+        System.out.println();
+    }
+
+    public void run() {
+        Scanner in = new Scanner(System.in);
+        String command;
+        while(in.hasNextLine()) {
+            command = in.nextLine();
+            if(command.equals("bye")) {
+                return;
+            } else if (command.equals("list")){
+                manager.listTask();
+            } else if (command.startsWith("done")){
+                manager.markTask(Integer.parseInt(command.substring(5)));
```

Comment:
You may want to consider creating a constant for the number 5 here. Creating a constant can help to give more meaning to the number written here. 

Reference Link: [Magic Number](https://nus-cs2113-ay1920s2.github.io/website/book/codeQuality/maximiseReadability/basic/avoidMagicNumbers)

Alternatively, you may want to consider changing this line to be more descriptive to avoid confusion, such as 
int taskNum = Integer.parseInt(command.substring(5)), or adding a comment to make it clearer what this does, like //command syntax: done 6